assets/src/edit-story/units/dimensions.js

```diff
@@ -72,10 +75,12 @@ export function dataToEditorX(x, pageWidth) {
  *
  * @param {number} y The value to be converted.
  * @param {number} pageHeight The basis value for the page's height in the "editor" space.
+ * @param {boolean} useFullbleed Whether the value is relative to the fullbleed or the safezone of the page.
  * @return {number} The value in the "editor" space.
  */
-export function dataToEditorY(y, pageHeight) {
-  return editorPixels((y * pageHeight) / PAGE_HEIGHT);
+export function dataToEditorY(y, pageHeight, useFullbleed = false) {
```

Comment:
I don't like it. We should really see if we can do it with one coordinate system.